cypress/local-only/tests/integration/caseDeadline/case-deadline-on-a-consolidated-group.cy.ts

```diff
@@ -0,0 +1,164 @@
+  };
+
+  before(() => {
+    cy.wrap({
+      leadDocketNumber: undefined,
+      memberDocketNumbers: [],
+    }).as('GROUP_INFO');
+
+    createAndServePaperPetition({
+      yearReceived: CASE_YEAR,
+    }).then(({ docketNumber: leadDocketNumber }) => {
+      loginAsDocketClerk1();
+      goToCase(leadDocketNumber);
+      updateCaseStatus(CASE_STATUS, JUDGE_NAME);
+      cy.get<GroupInfoType>('@GROUP_INFO').then(GROUP_INFO => {
+        GROUP_INFO.leadDocketNumber = leadDocketNumber;
+        cy.wrap(GROUP_INFO).as('GROUP_INFO');
+      });
+    });
+
+    for (let index = 0; index < 4; index++) {
+      createAndServePaperPetition({
+        yearReceived: CASE_YEAR,
+      }).then(({ docketNumber: memberDocketNumber }) => {
+        loginAsDocketClerk1();
+        goToCase(memberDocketNumber);
+        updateCaseStatus(CASE_STATUS, JUDGE_NAME);
+        cy.get<GroupInfoType>('@GROUP_INFO').then(GROUP_INFO => {
+          GROUP_INFO.memberDocketNumbers.push(memberDocketNumber);
+          cy.wrap(GROUP_INFO).as('GROUP_INFO');
+        });
+      });
+    }
+
+    cy.get<GroupInfoType>('@GROUP_INFO').then(GROUP_INFO => {
+      const { leadDocketNumber, memberDocketNumbers } = GROUP_INFO;
+      for (let index = 0; index < memberDocketNumbers.length; index++) {
+        const docketNumber = memberDocketNumbers[index];
+        goToCase(docketNumber);
+        cy.get('[data-testid="tab-case-information"]').click();
+        addCaseToGroup(leadDocketNumber!);
+      }
+    });
+  });
+
+  it('should add deadlines to all the children cases in the CG and bot update children removed from the group', () => {
+    loginAsDocketClerk1();
+
+    cy.get<GroupInfoType>('@GROUP_INFO').then(GROUP_INFO => {
+      const { memberDocketNumbers, leadDocketNumber } = GROUP_INFO;
+      goToCase(leadDocketNumber);
+      cy.get('[data-testid="case-detail-menu-button"]').click();
+      cy.get('[data-testid="menu-button-create-deadline"]').click();
+
+      cy.get('[id="deadline-date-picker"]').type('03/31/9999');
+      cy.get('[data-testid="case-deadline-description-input"]').type(
+        'John Is Testing',
+      );
+
+      cy.get('[data-testid="modal-button-confirm"]').click();
+      cy.get('[data-testid="success-alert"]').contains('Deadline saved.');
+      cy.get('[data-testid="tab-tracked-items"]').click();
+
+      cy.get('.case-deadline-row').should('have.length', 1);
+      cy.get('[data-testid="case-deadline-description"]').should(
+        'have.text',
+        'John Is Testing',
+      );
+
+      for (let index = 0; index < memberDocketNumbers.length; index++) {
+        const docketNumber = memberDocketNumbers[index];
+        goToCase(docketNumber);
+        cy.get('[data-testid="tab-tracked-items"]').click();
+        cy.get('.case-deadline-row').should('have.length', 1);
+
+        cy.get('[data-testid="case-deadline-description"]').should(
+          'have.text',
+          'John Is Testing',
+        );
+      }
```

Comment:
Minor, but I think we can do this all in the same loop.

Reply:
Meant to select the preceding `cy.get('[data-testid="tab-tracked-items"]').click();` too, but you get the idea.